spec/services/weather_man_spec.rb

```diff
@@ -11,20 +11,20 @@
 
   describe ".key_for" do
     it "returns a key for a given zip code and datetime" do
-      key = WeatherMan.key_for(10001, Time.zone.parse('January 1, 2015'))
-      expect(key).to eq "outdoor_temp_for_10001_on_2015-01-01"
+      key = WeatherMan.key_for(Time.zone.parse('January 1, 2015'), 'knyc')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.